index.html

```diff
@@ -15,11 +16,64 @@
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
-<body>
-  <header>
-  </header>
-  <main>
-  </main>
+
+<body class="background-lightgray">
+  <div data-view="entry-form">
+    <form id="form-journal">
```

Comment:
These two elements should be children of the `main` element, not the other way around